roles/tas_single_node/tasks/os.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: Fail if the OS is not RHEL
+  ansible.builtin.fail:
+    msg: "Unsupported OS. This playbook supports only RHEL."
```

Comment:
The message should say "role", not "playbook" to be precise. A playbook is something that the user would create and which would apply this role to a (set of) host(s). There are multiple occurences of this in this PR, please remember to fix them all.